app/bundles/FormBundle/Views/Builder/form.html.php

```diff
@@ -20,7 +20,8 @@
 <div id="mauticform_wrapper<?php echo $formName ?>" class="mauticform_wrapper">
     <form autocomplete="false" role="form" method="post" action="<?php echo $view['router']->url(
         'mautic_form_postresults',
-        ['formId' => $form->getId()]
+        ['formId' => $form->getId()],
+        self:ABSOLUTE_PATH
```

Comment:
Changing this to `\Symfony\Component\Routing\Generator\UrlGeneratorInterface::ABSOLUTE_PATH` instead would likely resolve the issue.

Reply:
https://github.com/mautic/mautic/issues/2778

Reply:
This is no longer the code in the latest version?

---

<div id="mauticform_wrapper<?php echo $formName ?>" class="mauticform_wrapper">
    <form autocomplete="false" role="form" method="post" action="<?php echo  $action; ?>" id="mauticform<?php echo $formName ?>" <?php if ($isAjax): ?> data-mautic-form="<?php echo ltrim($formName, '_') ?>"<?php endif; ?> enctype="multipart/form-data">
        <div class="mauticform-error" id="mauticform<?php echo $formName ?>_error"></div>
        <div class="mauticform-message" id="mauticform<?php echo $formName ?>_message"></div>
        <div class="mauticform-innerform">

            <?php
            /** @var \Mautic\FormBundle\Entity\Field $f */
            foreach ($fields as $fieldId => $f):
                if (isset($formPages['open'][$fieldId])):
                    // Start a new page
                    $lastFieldAttribute = ($lastFormPage === $fieldId) ? ' data-mautic-form-pagebreak-lastpage="true"' : '';
                    echo "\n          <div class=\"mauticform-page-wrapper mauticform-page-$pageCount\" data-mautic-form-page=\"$pageCount\"$lastFieldAttribute>\n";
                endif;

----

What to try and do with this then>